### PR TITLE
Fix JInputField component styles

### PR DESCRIPTION
**Pull request by flexxnn**

### CODE REVIEW

File: src/components/base/JInputField/JInputField.js [hunk 1]
```diff
@@ -71,6 +70,7 @@ export function JInputField({
       )}
     >
       <div
+        onClick={handleFocus(textInput)}
```
Comment (stcherenkov) [inline src/components/base/JInputField/JInputField.js]: Creates new function on every call to render. It will cause continuous re-renders, that could lead to performance issues or memory leaks. Please use class method, memoization or hook instead.

File: src/components/base/JInputField/jInputField.m.scss [hunk 2]
```diff
@@ -15,12 +15,7 @@
 .wrap {
   height: 72px;
   padding: 0;
-  box-shadow: $normal-shadow;
```
Comment (stcherenkov) [inline src/components/base/JInputField/jInputField.m.scss]: All inputs have shadow in design. Why did you remove it?

Comment (flexxnn) [reply src/components/base/JInputField/jInputField.m.scss]: This was not removed, it was moved to the color theme section, because we need to hide shadow in the disabled state
```
.white,
.white-indicator {
  .wrap {
    background-color: $white;	    background-color: $white;
    box-shadow: $normal-shadow;

     &:hover {
      box-shadow: $hover-shadow;
    }
  }
...
```

File: src/components/base/JInputField/jInputField.m.scss [hunk 3]
```diff
@@ -93,35 +86,40 @@
...
+    background: $white;
+
+    &:hover {
+      box-shadow: none;
+    }
 
     .label {
-      user-select: none;
       cursor: default;
+      color: $main-black-40;
```
Comment (IViol) [inline src/components/base/JInputField/jInputField.m.scss]: but it must be `$main-black-30`

Comment (IViol) [reply src/components/base/JInputField/jInputField.m.scss]: please recheck figma screens

Comment (flexxnn) [reply src/components/base/JInputField/jInputField.m.scss]: Checked and fixed
value + disabled - 30%
just disabled with value - 40%
